nepi/settings_production.py

```diff
@@ -31,6 +31,8 @@
 if 'migrate' not in sys.argv:
     INSTALLED_APPS.append('raven.contrib.django.raven_compat')
 
+LOCALE_PATHS = ('/var/www/nepi/meaningfulconsent/locale',)
```

Comment:
Not being terribly familiar with this project, what is the relationship between NEPI and meaningfulconsent, exactly?